pipelines/main/platforms/test_macos.soft_fail.arches

```diff
@@ -0,0 +1,11 @@
+# OS       TRIPLET                 ALLOW_FAIL    ARCH          MAKE_FLAGS     TIMEOUT
+macos      aarch64-apple-darwin    true          aarch64       .              .
+
+# These special lines allow us to embed default values for the columns above.
+# Any column without a default mapping here will simply substitute a `.` to the empty string
+
+# Of course we do not allow jobs to fail typically
+#default ALLOW_FAIL false
```

Comment:
Shouldn't we just drop `ALLOW_FAIL` completely and just set it to `true` when launching all `soft_fail.arches` files?

Reply:
Yes, that is much easier. I've implemented that change.